documentation/source/user_section/tutorials/1/1-5.rst

```diff
@@ -1,11 +1,15 @@
-.. _correcting_sct_propseg:
+1.5 Fixing a failed segmentation
+################################
 
-Correcting sct_propseg
-######################
+Due to contrast variations in MR imaging protocols, the contrast between the spinal cord and the cerebro-spinal fluid (CSF) can differ between MR volumes. Therefore, the propagated segmentation method may fail sometimes in presence of artifacts, low contrast, etc.
 
-What to do if the propagated segmentation fails or contains local errors?
+You have several options if the segmentation fails:
 
-Due to contrast variations in MR imaging protocols, the contrast between the spinal cord and the cerebro-spinal fluid (CSF) can differ between MR volumes. Therefore, the propagated segmentation method may fail sometimes in presence of artefact, low contrast, etc. Here above, we propose some protocols to correct some segmentation failures.
+- Manually correct the segmentation.
+- Try a different algorithm (``sct_deepseg_sc``).
+- Tweak the parameters of ``sct_propseg`` to suit your data.
+
+This page focuses on option 3 by providing some protocols to correct segmentation failures.
 
 .. contents::
    :local:
```

Comment:
You're right, this is totally unclear. It turns out the argument description is just as unclear, too:

https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/a685d7f0d8032f9df685a08d22947947ac416e71/spinalcordtoolbox/scripts/sct_propseg.py#L280-L285

I'll try to dig into the code to see what the actual behavior is (and try to clarify this in both places). Thank you for asking about this! :tada:

Reply:
Addressed here: https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3094/commits/b951cfacf8343be5742d1da68117c717ef6568e8 in `documentation/source/user_section/tutorials/spinalcord-segmentation/fixing-failed-segmentations.rst`

Explanation makes sense to me, thanks @joshuacwnewton !

Maybe a silly question, but I'm a bit confused about the image with the yellow circle? Is that where the segmentation is starting? Or where it broke?

Reply:
Not a silly question at all! In fact, I'm not actually sure of the answer myself....

(Just to give some context, I didn't write this page. It's actually a much older documentation page that was migrated as-is from [SCT's old SourceForge docs](https://sourceforge.net/p/spinalcordtoolbox/wiki/correction_PropSeg/). So, I think this may be the first time it's been properly reviewed.)

@jcohenadad Would you be able to clarify the intent of your figure here?